testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java

```diff
@@ -131,7 +133,7 @@ protected void start(PrintStream out) {
 
             commandBuilder.setJavaHome(legacyJavaHome == null ? javaHome : legacyJavaHome);
             if (jvmArgs != null) {
-                commandBuilder.setJavaOptions(jvmArgs.split("\\s+"));
+                commandBuilder.setJavaOptions(Stream.of(jvmArgs.split("\\s+")).filter(s -> !s.contains("agentlib:jdwp")).collect(Collectors.toList()));
```

Comment:
Please use a for loop.

This is only a test fixture but I don't want Stream API in our code if it's not required. It uses a lot more metaspace than a simple for loop and our code should send a consistent message that we strive to be optimal about metaspace.

@ctomc @jamezp please review in terms of the functional change.